ERCS/erc-7626.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 7626
```

Comment:
Numbers are assigned by ERC editors/associates.  Please don't self assign.

Reply:
Apologies for self-assigning the number. I'd like to confirm if the number 7626 assigned to this proposal has been accepted?